source/sass/main.scss

```diff
@@ -62,7 +65,6 @@
 @import "./views/youtube-regrets-reporter";
 @import "./views/dear-internet";
 @import "./views/publication";
-@import "./views/article";
```

Comment:
this is not itself a streamfield block, but a page type, so let's keep that in `views`, and only move the `// StreamBlocks` part to its own .scss file in the `blocks` dir.